hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileDeleteSuite.java

```diff
@@ -52,7 +51,6 @@ public List<HapiSpec> getSpecsInSuite() {
         return List.of(getDeletedFileInfo(), canDeleteWithAnyOneOfTopLevelKeyList());
     }
 
-    @HapiTest
```

Comment:
Another set of tests we need to fix, not disable.

Reply:
Okay, let's debug in another PR, to keep this one focused.